### PR TITLE
[WIP] Porting various proofs.

### PR DESCRIPTION
**Pull request by cangiuli**

### CODE REVIEW

File: path.red [hunk 1]
```diff
@@ -43,7 +88,43 @@ let trans
...
+let symmd
+  (A : Line^1 type)
+  (p : [i] A i with end)
+  : [i] (symm^1 type A) i with
+    | i=0 ⇒ p 1
+    | i=1 ⇒ p 0
+    end
+  =
+  λ i →
+    ?todo
```
Comment (cangiuli) [inline path.red]: I'm having difficulties here, if anyone wants to take a look.

**Comment by jonsterling** [comment]
Sorry for delay! I'm going to check this out soon. Just been very busy (and will continue to be busy) with FLOC/LICS preparations.

File: path.red (hunk 1 above)
Comment (jonsterling) [reply path.red]: Apparently, this is fixed in my branch. I'm going to merge your code into my branch and then it will be merged together.

**Comment by jonsterling** [comment]
Merged as part of my big PR.

**Pull request closed without merging**